src/modules/landing_target_estimator/landing_target_estimator_main.cpp

```diff
@@ -140,7 +140,7 @@ int landing_target_estimator_thread_main(int argc, char *argv[])
 
 	while (!thread_should_exit) {
 		est.update();
-		px4_usleep(1000000 / landing_target_estimator_UPDATE_RATE_HZ);
+		px4_usleep(1_s / landing_target_estimator_UPDATE_RATE_HZ);
```

Comment:
not directly related to your PR, but while we're here -- @ndepal just checking that the automatic flooring by the integer division is what you intended here?

Reply:
Yes, flooring is acceptable assuming `1_s` is still treated as 1000000 during the division